client/src/index.tsx

```diff
@@ -25,14 +26,13 @@ const root = createRoot(container);
 
 root.render(
   <React.StrictMode>
-		<GlobalStyles />
-    <Provider store={store}>
-      <App />
-    </Provider>
+    <BrowserRouter>
+      <GlobalStyles />
+      <Provider store={store}>
+        <App />
+      </Provider>
```

Comment:
En esta parte del codigo es que se debe de añadir el ThemeProvider de style-components.

```suggestion
import Theme from "./Theme/Theme" // O sea la que sea la ruta de Theme.ts
import { ThemeProvider } from "styled-components"
// [...]
      <GlobalStyles />
      <ThemeProvider theme={theme}>
        <Provider store={store}>
          <App />
        </Provider>
      </ThemeProvider>
```